lighthouse-cli/cli-flags.js

```diff
@@ -171,6 +185,29 @@ function getFlags(manualArgv) {
           'For more information on Lighthouse, see https://developers.google.com/web/tools/lighthouse/.')
       .wrap(yargs.terminalWidth())
       .argv;
+
+  // ".middleware" does not exist in this version of yargs, so do some post-processing here.
```

Comment:
```suggestion
  // Support comma-separated values for some array flags by splitting on any ',' found.
```
(fine to be specific and we can generalize it if we add more post-processing in the future)